addons/html_editor/static/src/main/chatgpt/language_selector.xml

```diff
@@ -13,8 +13,8 @@
             </button>
             <t t-set-slot="content">
                 <t t-foreach="state.languages" t-as="language" t-key="language[0]">
-                    <DropdownItem onSelected="() => this.onSelected(language[1])">
-                        <div class="user-select-none lang" t-esc="language[1]"/>
+                    <DropdownItem class="'user-select-none'" onSelected="() => this.onSelected(language[1])">
```

Comment:
I'm very confused because all those have double quote followed by simple quotes. Does it even work? I mean, the effect of the class. I just tested and `class="'my-class'"` does not seem to activate the style from `.my-class`.

Reply:
Here class term is a prop for the `DropdownItem` component so we need to pass it as a string.

Reply:
Is this something we control that we can rename or is it from web?

Reply:
We can't its from web see https://github.com/odoo/odoo/blob/18.0/addons/web/static/src/core/dropdown/dropdown_item.js#L13 and https://github.com/odoo/odoo/blob/18.0/addons/web/static/src/core/dropdown/dropdown_item.xml#L8